src/lib/Scenes/Artwork/Components/SellerInfo.tsx

```diff
@@ -9,11 +9,9 @@ interface SellerInfoProps {
 
 export class SellerInfo extends React.Component<SellerInfoProps> {
   render() {
-    return (
-      <Box>
-        <Sans size="3">{this.props.artwork.partner.name}</Sans>
-      </Box>
-    )
+    const { partner } = this.props.artwork
+    const { name } = partner
+    return <Box>{name && <Sans size="3">{name}</Sans>}</Box>
```

Comment:
Instead of returning an empty Box when the name isn't there, can we check for `name` before rendering anything? Not sure if there's some hidden reason for doing it the current way.

Reply:
I'm assuming there's going to be more rendered in this component and just checking for the partner name for now, the component shouldn't be rendered at all if the relevant data is missing so the box wont be rendered in that case either way.

Reply:
Drive-by FYI: if at some point you need more than, let’s say, two fields to determine if a section should be shown and that can semantically accurately be captured as a single predicate field in MP, then moving it there may become preferable.